inst/include/interface/rcpp/rcpp_objects/rcpp_interface_base.hpp

```diff
@@ -67,48 +94,72 @@ class Parameter {
 
 uint32_t Parameter::id_g = 0;
 
+std::ostream& operator<<(std::ostream& out, const Parameter& p) {
+    out << "Parameter:{" << "id:" << p.id_m << ",\nvalue:" << p.value_m
+            << ",\nestimated_value:" << p.estimated_value_m << ",\nmin:"
+            << p.min_m << ",\nmax:" << p.max_m << ",\nestimated:" << p.estimated_m << "\n}";
+    return out;
+}
```

Comment:
@msupernaw this information on line 97--102 is printed every time a value is assigned to a parameter. Was that intentional? See below for an example. I noticed this when running the vignette locally.
![image](https://github.com/user-attachments/assets/ce821f44-16cc-42ba-8704-2a6d12952329)

Reply:
There were cout statements left that was causing the printing this has been fixed.